tools/src/test/resources/com/nedap/archie/diff/specexamples/openEHR-EHR-CLUSTER.lipid_studies_panel.adls

```diff
@@ -13,7 +13,6 @@ description
 definition
     CLUSTER[id1.1] matches {    -- Lipid studies panel
         items matches {
-            after [id3]
```

Comment:
In the flattener, specialised nodes will appear after their parent if there is no sibling order attacked to the specialised nodeId